j/jackson-modules-java8/build-info.json

```diff
@@ -7,7 +7,7 @@
     "validate_build_script": "true",
     "use_non_root_user": "false",
 
-    "jackson_module_java8_ubi_8_5.sh": {
+    "jackson-modules-java8-2.14.1": {
```

Comment:
@stutiibm Have you validated build script for multiple version. If it is, we will need to add some pattern here to make this json compatible for multiple version.